infer/tests/build_systems/jar/issues.exp

```diff
@@ -0,0 +1,3 @@
+/home/jeremy/Code/infer/infer/tests/build_systems/codetoanalyze/mvn/simple_app/src/main/java/com/mycompany/Hello.java, hello.Hello.mayCauseNPE():void, 4, NULL_DEREFERENCE, no_bucket, ERROR, [start of procedure mayCauseNPE(),Skipping Random(): unknown method]
+/home/jeremy/Code/infer/infer/tests/build_systems/codetoanalyze/mvn/simple_app/src/main/java/com/mycompany/Hello.java, hello.Hello.mayLeakResource():void, 7, RESOURCE_LEAK, no_bucket, ERROR, [start of procedure mayLeakResource(),Taking false branch]
+/home/jeremy/Code/infer/infer/tests/build_systems/codetoanalyze/mvn/simple_app/src/main/java/com/mycompany/Hello.java, hello.Hello.twoResources():void, 15, RESOURCE_LEAK, no_bucket, ERROR, [start of procedure twoResources(),Taking true branch,exception java.io.IOException]
```

Comment:
This contains absolute paths. Maybe just a matter of setting the project root to the test directory when running infer in the test?